lib/api/client-commands/registerBasicAuth.js

```diff
@@ -0,0 +1,50 @@
+const ClientCommand = require('./_base-command.js');
+const {Logger} = require('../../utils');
+
+/**
+ * Automate the input of basic auth credentials whenever they arise.
+ *
+ * @example
+ *  this.demoTest = function (browser) {
+ *    browser.registerBasicAuth(1000, 800);
+ *    browser.url('https://page-needs-auth-credentials.com');
```

Comment:
we can use a proper url as example so that users can try it out. E.g. `[browserspy.dk](http://browserspy.dk/password-ok.php)` which is used here: https://crossbrowsertesting.github.io/